### PR TITLE
Provide API endpoint for operations

### PR DESCRIPTION
**Pull request by dougestey**
So I'm extremely rusty with PHP and have no experience with Laravel whatsoever. That said, I've written a simple endpoint for SeAT's Swagger API that exposes all operations. It's been useful for my purposes, and might be for others too.

Issues with this PR:
- Still not showing up in Swagger documentation (might need some help with this)
-

### CODE REVIEW

**Comment by warlof** [comment]
Eyah, thank you a lot for the contribution :)

Regarding your Swagger issue, you have to update the Provider and include this small tweak inside https://github.com/eveseat/web/blob/master/src/WebServiceProvider.php#L385-L402

This will add the directory from the plugin into the Swagger documentation discoverying directories.

**Comment by dougestey** [comment]
Thanks @warlof - that did the trick. :)

File: src/Http/Controllers/Api/v1/CalendarApiController.php [hunk 1]
```diff
@@ -0,0 +1,59 @@
...
+     *     ),
+     *     @SWG\Response(response=400, description="Bad request"),
+     *     @SWG\Response(response=401, description="Unauthorized"),
+     *    )
+     *
+     * @return \Illuminate\Http\Resources\Json\AnonymousResourceCollection
+     */
+    public function getAllOperations()
+    {
+        $operations = Operation::all();
```
Comment (warlof) [inline src/Http/Controllers/Api/v1/CalendarApiController.php]: Could you use the Laravel Resource and paginate result in order to return response ?
So the pagination is managed by the Framework based on request parameters :)

You got some sample in core like the endpoint used to return Character Assets
https://github.com/eveseat/api/blob/master/src/Http/Controllers/Api/v2/CharacterController.php#L210-L211

Comment (dougestey) [reply src/Http/Controllers/Api/v1/CalendarApiController.php]: Will do :)

**Comment by dougestey** [comment]
Will get to this in the next few days. Sorry for the delay!

**Pull request closed without merging**